src/agents/workflows/review_logic.py

```
@@ -36,59 +71,215 @@ def _ensure_dict_review_state(review_state) -> Dict[str, Any]:
+            return str(content or "")
+    return None
+
+
+def safe_invoke_model(model, prompt, timeout=15, retries=2) -> Any:
+    """모델 호출 시 타임아웃 및 재시도 로직 적용. 반환값은 모델 설정에 따라 다름."""
+    last_exc = None
+    for attempt in range(retries + 1):
+        try:
+            # LangChainChat 모델의 invoke 호출
+            return model.invoke(prompt, timeout=timeout)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)